src/main/java/com/google/api/codegen/py/PythonContext.java

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.py;
+
+/**
+ * A PythonContext provides functionality specific to a context in Python.
+ */
+public interface PythonContext {
+  void setPythonSnippetSet(PythonSnippetSet<?> pythonSnippetSet);
```

Comment:
I'll take a look and see how that was done. The call path is not exactly the same is in Python, because so much more of writing Python docstrings is done in the toolkit Java code, rather than in the template code.